tests/verus-consistency.rs

```diff
@@ -1093,17 +1099,25 @@ fn try_parse() -> (u: u32)
 
     impl a {
         // My favorite function
-        fn b()  //
+        fn b()
+        //
         ;
     }
 
-    fn try_parse() -> (u: u32)  //    1
-      //    2
-      //    3
+    fn try_parse() -> (u: u32)
+    //    1
+    //    2
+    //    3
     {
         8
     }
 
+    pub fn test() {
+        if false {
+        }  // No space between the one character indicating non-inline and the comment
+
+    }
```

Comment:
The extra newline at the end is non-ideal; I messed around with it a little bit to try to convince it into not placing the extra newline, but it seems we'll need to include extra tracking in the `Context` to handle this, so I dropped looking at it further.

I expect this particular situation to be not very common though, and it handles the more common things like below just fine, so I think we just keep this as is, and shipit
```rs
pub fn test() {
    if false {
        ();  // Never going to take that branch
    }
}
```